src/DashboardService.Models.Dto/Requests/Comment/CreateCommentRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Comment;
+
+public record CreateCommentRequest
+{
+  public Guid TaskId { get; set; }
+  public Guid CreatedBy { get; set; }
+  public DateTime CreatedAtUtc { get; set; }
```

Comment:
this comes from session. Remove it from request

Reply:
++